Jenkinsfile

```diff
@@ -501,8 +498,6 @@ def runScript(Map args = [:]){
             sh(label: "Run tests: Elastic Stack ${stack} - ${scope} - ${env.MODE}", script: '.ci/scripts/test.sh')
           }
         }
-      } catch(e) {
-        throw e
```

Comment:
question: do we swallow errors now or is it handled somewhere else?

Reply:
`try/finally` works fine as long as the body in the `try` closure can handle the exception. In this case, I decided to simplify the try/catch/finally since the `catch` does nothing but throwing the error, so it's a bit more idiomatic, therefore this should not shallow any errors